test/integration/features/basic.feature

```diff
@@ -0,0 +1,20 @@
+# 
+Feature: Bringup
+  In order to use MiniShift
+  As a user
+  I need to be able to bring up a test environment
+
+  Scenario: Docker environment variables
+    Given a "developer"
+     When executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT"
+      And should be running
+     Then should have a valid IP address
+     Then executing "minishift ssh echo hello" should contain
+      """
+      hello
+      """
+     Then executing "minishift ssh cat /var/lib/boot2docker/profile" should contain
```

Comment:
Personal preference as I do not prefer to keep state of a previous action, unless it is stored inside the application/context and there is a verifiable action from the applications interface.

And yes, it currently implements/expects the happy-flow.